src/args.ts

```diff
@@ -18,14 +16,19 @@ type ImportSpecificParams = {
 };
 
 type ExportSpecificParams = {
+  format: 'yaml' | 'directory';
   output_folder: string;
-  export_ids: boolean;
+  export_ids?: boolean;
 };
 
 export type ExportParams = ExportSpecificParams & SharedParams;
 export type ImportParams = ImportSpecificParams & SharedParams;
 
-function getParams(): ExportParams | ImportParams {
+export type CliParams = (ExportParams | ImportParams) & {
+  _: ['export' | 'import' | 'deploy' | 'dump'];
```

Comment:
This underscore property is only useful in the context of the CLI and has no associated function for module import use cases.